test/plugins/node-workspace.ts

```diff
@@ -230,10 +255,10 @@ describe('NodeWorkspace plugin', () => {
       );
       expect(nodeCandidate).to.not.be.undefined;
       const updates = nodeCandidate!.pullRequest.updates;
-      assertHasUpdate(updates, 'package.json');
-      assertHasUpdate(updates, 'node1/package.json');
-      assertHasUpdate(updates, 'node4/package.json');
       snapshot(dateSafe(nodeCandidate!.pullRequest.body.toString()));
+      snapshotUpdate(updates, 'package.json');
```

Comment:
curious why this order changed.

Reply:
The snapshot name is based on an incrementing count. To make the snapshots only additive, I put these ones after the existing snapshot.